Makefile

```diff
@@ -1,8 +1,2 @@
 style:
 	poetry run black .
```

Comment:
Hey, thanks for the contribution! This looks confusing, though. Let's stand with just one Makefile, it's better for us to focus all resources on a single place, so we make it easier to contribute.

Reply:
Lets keep all of the instructions inside `src/Makefile`

Reply:
Yes, at first I thought it could be useful to keep other tasks in a root Makefile, but it makes sense to keep only one inside `src/Makefile`. I'll fix that.